osu.Game/Screens/Play/HUD/HitErrorDisplay.cs

```diff
@@ -53,12 +74,15 @@ private void onNewJudgement(JudgementResult result)
                 return;
 
             foreach (var c in Children)
-                c.OnNewJudgement(result);
+            {
+                foreach (var meter in c.Children)
+                    meter.OnNewJudgement(result);
+            }
```

Comment:
I'm not too hot on this. Can we not reuse meter instances on both sides and just update the single ones?

Reply:
IMO this looks readable and can avoid duplicating code

Reply:
I meant the double update - if you have one meter type on both sides you get two instances that update independently. Could be nice to have just one, but displayed in two places, but that's a nitpick really.